<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-P7KUTMDR" ignored="0" name="FIREFOXtest" tests="1" failures="1" timestamp="11 Nov 2019 15:19:42 GMT" time="80.919" errors="0">
  <testcase name="testVerifyInfoProducSearchVsCartScreens" time="19.467" classname="com.flipkart.tests.VerifyInfoProducSearchVsCartScreens">
    <failure type="java.io.FileNotFoundException" message=".\data\input.xlsx (The process cannot access the file because it is being used by another process)">
      <![CDATA[java.io.FileNotFoundException: .\data\input.xlsx (The process cannot access the file because it is being used by another process)
at java.io.FileOutputStream.open0(Native Method)
at java.io.FileOutputStream.open(Unknown Source)
at java.io.FileOutputStream.<init>(Unknown Source)
at java.io.FileOutputStream.<init>(Unknown Source)
at com.flipkart.tests.VerifyInfoProducSearchVsCartScreens.testVerifyInfoProducSearchVsCartScreens(VerifyInfoProducSearchVsCartScreens.java:86)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testVerifyInfoProducSearchVsCartScreens -->
</testsuite> <!-- FIREFOXtest -->
